src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.OSX.cs

```diff
@@ -13,8 +13,11 @@ public partial class ProcessThread
         /// </summary>
         private ThreadPriorityLevel PriorityLevelCore
         {
-            get { throw new PlatformNotSupportedException(); }
-            set { throw new PlatformNotSupportedException(); }
+            // We couldn't find a POSIX API on OSX
```

Comment:
You can probably use // OSX does not provide a way to get this data like below..